pkg/events/events.go

```diff
@@ -5737,7 +5740,7 @@ var Definitions = eventDefinitions{
 			Dependencies: dependencies{
 				Events: []eventDependency{{EventID: SecurityMmapFile}},
 			},
-			Sets: []string{"lsm_hooks", "fs", "fs_file_ops", "proc", "proc_mem"},
+			Sets: []string{"lsm_hooks", "fs", "fs_file_ops", "proc", "proc_mem", "higher_level"},
```

Comment:
The name of the set "higher_level" is a bit vague.
Maybe rename to "derived"? Other suggestions?

Reply:
I think derived implies other derived events we have which aren't for the same purpose, like the `hooked` family of events, which are more logical extensions of "alerting" events where as this is a simplified version of an existing event.
I think "simple" could be a possible name for the set and we could have more non derived events in it.

Reply:
How about "monitor" or "audit"?

Reply:
"simple" is an option, although still vague.
"monitor"/"audit" - doesn't it fit many other events as well?

maybe "abstract" or "user_friendly"?
In the future, we may want to make this the "default" set of tracee, instead of the existing default set

Reply:
I like user_friendly